src/assets/i18n/cs.json5

```diff
@@ -656,7 +656,7 @@
   // "clarin.license.agreement.signer.header.info": ["The information below identifies you, the SIGNER. If the information is incorrect, please contact our", "Help Desk.", "This information will be stored as part of the electronic signatures."],
   "clarin.license.agreement.signer.header.info": ["Informace uvedené níže vás, PODEPSANÉHO, identifikují. Pokud jsou tyto informace nesprávné, kontaktujte prosím", "poradnu.", "Tyto informace budou uloženy spolu se záznamem o souhlasu."],
   // "clarin.license.agreement.item.handle": "Item handle",
-  "clarin.license.agreement.item.handle": "Klika položky",
+  "clarin.license.agreement.item.handle": "Handle položky",
```

Comment:
PID? Identifikator? Musi to byt vzdy Handle?

Reply:
Zatiaľ tam zobrazujeme vždy Handle, ak by to mohol byť aj DOI, tak by trebalo trochu pomenit veci. Každý Item ma vždy nejaký Handle, tak ideálna oprava by bola, ak by zobrazilo aj DOI popri Handle, ak ma Item DOI.